definitions/json/ComplexTypes/ComplexTypes-ET3.json

```
@@ -1165,5 +1165,15 @@
     "RetainHiddenValue": "Yes",
     "SecurityClassification": "Public",
     "Searchable": "N"
+  },
+  {
+    "ID": "Respondent",
+    "ListElementCode": "et3NotificationAcceptedDate",
+    "FieldType": "Text",
+    "ElementLabel": " ",
+    "FieldShowCondition": "responseReceived=\"dummy\"",
+    "RetainHiddenValue": "Yes",
+    "SecurityClassification": "Public",
+    "Searchable": "N"
```

This one does need to be searchable as ACAS needs this field
```suggestion
    "Searchable": "Y"
```